packages/components/src/List/DisplayTable/DisplayTable.component.js

```diff
@@ -149,7 +149,9 @@ function getNextDirection(isCurrentSortField, currentSort) {
 export function ColumnHeader({ index, column, sort }) {
 	let header;
 
-	if (sort) {
+	if (column.hideHeader) {
```

Comment:
The goal of this property is to keep the screen reader capability for the columns with no header. For example, an "Actions" columns as it is in TDP's datasets listing page should have an hidden header but a screen reader must be able to read the content.

In addition, passing an empty label in a table which have sorting capabilities will render an empty button which will be focusable by pressing the 'tab' key and create some issues as described here : https://jira.talendforge.org/browse/TDP-3549